src/mlpack/core/data/image_info.hpp

```diff
@@ -22,7 +22,8 @@ namespace data {
 /**
  * Checks if the given image filename is supported.
  *
- * @param filename Name of the image file.
+ * @param fileName Name of the image file.
+ * @param save Set to true to check if the filename can be saved, else loaded.

```

Comment:
```suggestion
 * @param save Set to true to check if the file format can be saved, else loaded.
```
Just a little bit of extra clarity---this doesn't actually check whether permissions on the directory are ok for file saving or anything, just that the format (specified by the extension) is save-able. (or... load-able)